ONSdigital/census-rm-acceptance-tests-2021
# Use the current context of kubectl if no ENV is provided

# Motivation and Context

@davidmort pointed out that the script should default to using the current kubectl context if ENV variable is omitted when calling the run_gke.sh script.

# What has changed



- Use the current context of kubectl if no ENV is provided
- Set the namespace if provided
- Remove default namespace (following https://github.com/ONSdigital/census-rm-terraform/pull/34)
- Update readme and print lines

# How to test?



To run the acceptance tests (`latest` image in CI GCR) in a pod within the current k8s context:

- Create and [terraform apply](https://github.com/ONSdigital/census-rm-terraform) a new GCP project or an existing RM dev project, e.g.: `census-rm-attest23`
- Once the `apply.sh` script has finished pushing the services into RM and everything is green`*` the kubectl context should then be pointing at the RM cluster:
```bash
./run_gke.sh
```

`*` If the actionexportersvc is failing, connect to the Cloud SQL and drop and re-create the pgcrypto extension.
